### PR TITLE
Update test-infra

### PR DESCRIPTION
**Pull request by adrcunha**
Major change: use standard presubmit test runners.

Last PR that fixes knative/test-infra#170

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -29,8 +29,7 @@ source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.s
 
 # Script entry point.
 
-initialize $@
-
-# TODO(#8): Write E2E tests.
+echo "TODO(#8): Write E2E tests."
+# initialize $@
```
Comment (evankanderson) [inline test/e2e-tests.sh]: Is this `initialize` still valid once a test is present? Is it harmful if we call `initialize $@` with no e2e tests at the moment?

Comment (adrcunha) [reply test/e2e-tests.sh]: It's not harmful, but you're wasting time and resources creating a cluster that's not used. I'll add a comment clarifying.

Comment (adrcunha) [reply test/e2e-tests.sh]: Actually I just removed the commented code.

**Comment by adrcunha** [comment]
> I'm presuming this will still create the releases from the files in `contrib/`?

The behavior of `release.sh` didn't change, just its format.

**Pull request merged**